apps/server/src/modules/authorization/types/rule.interface.ts

```diff
@@ -1,8 +1,8 @@
-import { User } from '@shared/domain';
-import { AuthorizableObject } from './authorizable-object.type';
+import { BaseDO, User } from '@shared/domain';
+import { AuthorizableObject } from '@shared/domain/domain-object'; // fix import when it is avaible
```

Comment:
These two imports could be combined to 
```
import { AuthorizableObject, BaseDO, User } from '@shared/domain';
```
Or do you want to keep it for the sake of the comment?

It seems there are also other instances, where the import of `AuthorizableObject` may be combined with others.

Reply:
Das geht leider aktuell nicht. Ich musste die Änderungen wieder reverten. Es müssen einige import / exports + nest module Sachen glatt gezogen werden. Das habe ich in ein follow up ticket ausgelagert. Muss aber noch mit Details ausgefüllt werden.